python/langsmith/evaluation/_runner.py

```diff
@@ -1653,13 +1662,14 @@ def _get_run(r: rt.RunTree) -> None:
 
 
 def _resolve_data(
-    data: DATA_T, *, client: langsmith.Client
+    data: DATA_T, *, client: langsmith.Client,
 ) -> Iterable[schemas.Example]:
     """Return the examples for the given dataset."""
+    # TODO: Find a smarter way of determining whether to get attachments (don't just default to true)
```

Comment:
would appreciate suggestions. I don't want to pass the target function to the ExperimentManager, but not sure what the best way to do this is